docs/developer-guides/getting-started.md

```diff
@@ -0,0 +1,121 @@
+# Getting started
+
+This is a guide for partner chains developers to get started in the project. When you complete it, you should be able to build the project, test locally, and access development environment hosted in EKS.
+
+Prerequisites:
+* `nix` installed
+* `docker` installed
+* `protoc` installed [link](https://google.github.io/proto-lens/installing-protoc.html)
```

Comment:
Does anyone remember why we need protoc installed explicitly, and not provided by the devshell? @LGLO

Reply:
I think this is added by @paul-weiss to give some way of proceeding without nix, but I'm not really sure.

I don't have `protoc` installed system wide or for my user, so I certainly use one from devshell.